rules-topics/01_introduction.md

```diff
@@ -62,6 +62,56 @@ Welcome to Chronicles. Whether you've been LARPing for years, or have never seen
 tabard in your life, Chronicles welcomes you. For the newer players, here are a few tips
 to get you started
 
+### Creating a Character
+
+If you are a new player you will start with 100 Skill Points 10 Body and 5 Armor Training.  Skill Points are used to purchase skills like the ability to swing a sword or cast a spell.  You can use the Character Calculator to design your character.  
+* All characters earn 3 Skill Points per day of an event.
+* Characters Under 200 Build gain Skill Points at 2x the rate.
+* All Characters under 200 build can be recreated or rewritten at any time.
+* You can accrue up to 1 extra Skill Point per day by working a NPC shift (See the section on Goblin Points for more details).
+
+If you have a character in another Larp System please see our Transfer Policy if you want to transfer your character to Chronicles.
+
+### Secondary Characters 
+Once you hit 200 Skill Points you can create a secondary character.  This character will always have 75 percent of your primary character’s skill points.  If you play a secondary character, then all gained SP will be applied to the primary character.  Strain is tracked separately on each character.
+
+If you wish to create an additional character beyond a secondary character you can.  All characters beyond the second character will have 50 percent of the primary character’s Skill Points.  All gained SP will be applied to the primary character.  Strain is tracked separately on each character.
+
+### Identify Ability
+Allows the character to Identify an item, a group of items, or target after 1 minute of concentration and continuous contact. This will make the nature and workings of enchantments on an item/creature known to the character, at the discretion of plot. There may be some properties that still remain hidden to the character.
+
+Status that require Identify:
+|Identify Needed|
+|:---:|
+|Animate|
+|Curse|
+|Curse of Transformation|
+|Infect|
+|Spirit Bottle|
+
+### Floating Reset
+Some skills or abilities are listed as "Daily".  These skills and abilities are renewed between 4:00 and 6:00 PM.  If you are on a module between that timeframe you must attempt the reset immediately after a mod.  Daily skills and abilities require 10 minutes of concentration/Rest to refresh.
+
+### How Does Combat Work
+
+Chronicles is a boffer combat Larp that focuses as much on roleplay and interaction between PCs and NPCs as physical combat.  We emphasize light combat, and a general rule is if you swing enough for it to hurt someone you are probably swinging too hard.  The goal of combat is to be safe and while accidents may happen you are more likely to twist an ancle on uneven ground than hurt yourself during combat.  More details can be found in the Combat section of the rulebook.
```

Comment:
"Larp" to "LARP" please?

Suggestion for second & third sentences: `The goal of combat is for all participants and observers to be safe.  We emphasize light combat.  A general rule is, if your swing is enough to hurt, someone, you are probably swinging too hard.  While accidents may happen, you are more likely to twist an ankle on uneven ground than hurt yourself during combat.`

Reply:
Fixed